package.json

```diff
@@ -4,35 +4,20 @@
     "scripts": {
         "dev": "vite",
         "build": "vite build",
-        "format": "prettier --write . && prettier --write **/*.blade.php",
-        "format:check": "prettier --check . && prettier --check **/*.blade.php"
+        "format": "npx prettier --write .",
+        "format:check": "npx prettier --check ."
```

Comment:
For some reason with the other prettier-plugin-blade setup, I had to run blade files separately. Now I am getting them all in one go.